package.json

```diff
@@ -21,6 +21,7 @@
     "find-translations": "node --loader ts-node/esm util/find_missing_translations.js"
   },
   "devDependencies": {
+    "@babel/types": "7.12.*",
```

Comment:
7.13.x breaks current code.

Reply:
Wait, the newer version breaks current code? What is the issue?

Reply:
It breaks the function `getAllKeys` wroten by @MaikoTan , I didn't dig into details

Reply:
Okay, i'll try to figure it out when I have time.